sidecar/config.go

```diff
@@ -333,7 +337,7 @@ func (cfg *Config) buildXenonConf() []byte {
 			"admin": "root",
 			"ping-timeout": %d,
 			"passwd": "%s",
-			"host": "localhost",
+			"host": "127.0.0.1",
```

Comment:
root@127.0.0.1 should  set empty password

Reply:
127.0.0.1 and localhost with sample password now.